app/scripts/controllers/buildConfig.js

```diff
@@ -65,6 +66,40 @@ angular.module('openshiftConsole')
               });
             }
 
+            $scope.updatedBuildConfig = angular.copy($scope.buildConfig);
+            $scope.buildStrategy = $filter('buildStrategy')($scope.updatedBuildConfig);
```

Comment:
actually, does this even need to be on the scope object, the only references to buildStrategy in the views that i am seeing are using the filter, not this object